src/utils/preview.ts

```diff
@@ -0,0 +1,16 @@
+import * as vscode from "vscode";
+
+export async function preview(uri: vscode.Uri, column: vscode.ViewColumn, tabTitle: string): Promise<void> {
+    const html = await getHTML(uri);
+    const w = vscode.window.createWebviewPanel('vsk8s-preview', tabTitle, column, {
```

Comment:
as above wrt singleton

Reply:
Definitely not in this case, because a user might very well want to have two previews up at the same time.  This is shared infrastructure between multiple commands so even if one command wanted to reuse its webview, that should not leak across to sharing with other commands.